src/lib/server/eliommod_pagegen.ml

```diff
@@ -226,17 +227,19 @@ let execute
       user_tab_cookies) as info)
     sitedata =
 
-  Lwt.catch
-    (fun () -> generate_page now info sitedata)
-    (fun e -> handle_site_exn e info sitedata)
-  >>= fun result ->
-  update_cookie_table ~now sitedata
-    ((service_cookies_info, data_cookies_info, pers_cookies_info), secure_ci)
-  >>= fun () ->
-
-  update_cookie_table ~now sitedata
-    ((service_tab_cookies_info, data_tab_cookies_info, pers_tab_cookies_info), secure_ci_tab)
-  >>= fun () ->
+  let* result =
```

Comment:
Did we choose between `let*` and `lwt`?

Reply:
Oh, for some reason I thought that we didn't use `lwt_ppx` in eliom.
Here is a PR replacing all `let*` by `let%lwt` https://github.com/ocsigen/eliom/pull/741.